kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/S3SinkConnectorIT.java

```diff
@@ -241,7 +265,8 @@ public void testFilesWrittenToBucketJsonWithExtInTopic() throws Throwable {
    */
   private void testBasicRecordsWritten(
           String expectedFileExtension,
-          boolean addExtensionInTopic
+          boolean addExtensionInTopic,
+          boolean verifyContinuumNotificationOffsets
```

Comment:
Should we add this to the javadoc above?

Reply:
Did so.